addons/website_blog/models/website_blog.py

```diff
@@ -257,6 +257,13 @@ def _notify_record_by_inbox(self, message, recipients_data, msg_vals=False, **kw
     def _default_website_meta(self):
         res = super(BlogPost, self)._default_website_meta()
         res['default_opengraph']['og:description'] = res['default_twitter']['twitter:description'] = self.subtitle
+        res['default_opengraph']['og:type'] = 'article'
+        res['default_opengraph']['article:published_time'] = self.post_date
+        res['default_opengraph']['article:modified_time'] = self.write_date
+        res['default_opengraph']['article:author'] = self.author_id.sudo().name
+        res['default_opengraph']['article:section'] = self.subtitle
+        if self.tag_ids:
+            res['default_opengraph']['article:tag'] = self.tag_ids[0].name
```

Comment:
Why only the first? We could use all as stated in the doc, no?
`article:tag - string array - Tag words associated with this article.`

Reply:
It is a dict so we can only use the key value once, unless we change the whole logic of how this is built, right?